dapps/templates/demo/app/components/whisper.js

```diff
@@ -83,9 +85,11 @@ class Whisper extends React.Component {
               defaultValue={this.state.listenTo}
               placeholder="channel"
               onChange={e => this.handleChange(e, 'listenTo')}/>
-            <Button bsStyle="primary" onClick={(e) => this.listenToChannel(e)}>Start Listening</Button>
+            <Button disabled={!this.state.channelIsValid} bsStyle="primary" onClick={(e) => this.listenToChannel(e)}>Start Listening</Button>
```

Comment:
Please add a message when `!channelIsValid` saying that you need a length of more than 3. Otherwise, it's confusing why the button stays gray.

Reply:
Done